stubs/Send2Trash/send2trash/util.pyi

```diff
@@ -1 +1,4 @@
-def preprocess_paths(paths): ...
+from _typeshed import StrOrBytesPath
+from collections.abc import Sequence
+
+def preprocess_paths(paths: StrOrBytesPath | Sequence[StrOrBytesPath]) -> list[str | bytes]: ...
```

Comment:
It checks for `list` with isinstance(), so we need `list[StrOrBytesPath]`. Except that it doesn't work because of invariance, so we need `list[Any]`. I think it's more important to guard against passing a non-`list` sequence than passing a list with the wrong element type.

In any case, this should be made consistent with `__init__.pyi`.

Reply:
I went with Sequence to avoid the invariance, but don't have a strong opinion on that vs. `list[Any]`.